docs/sources/setup-grafana/configure-security/configure-authentication/jwt/index.md

```diff
@@ -183,7 +187,8 @@ key_id = my-key-id
 
 By default, only `"exp"`, `"nbf"` and `"iat"` claims are validated.
 
-You might also want to validate that other claims are really what you expect them to be.
+You might also want to validate that other claims are really what you expect them to be, which you can do by setting the `expect_claims` configuration option.
```

Comment:
Consider validating that other claims match your expectations by using the `expect_claims` configuration option.